blog/_posts/2021-10-19-scala-3.1.0-released.md

```diff
@@ -6,7 +6,14 @@ title: Scala 3.1.0 released!
 ---
 Hello from the Scala 3 team! It has already been six weeks since we have announced the release candidate for the first minor version after the initial release of Scala 3. Now, after two more RCs, we can confidently promote Scala 3.1.0-RC3 to a new stable release of the language. In other words, Scala 3.1.0 is officially out!
 
-Thanks to Scala 3 strong compatibility guarantees, all code working in 3.0.2 will also work in 3.1. On the other hand, switching to Scala 3.1 allows you to use many improvements and a handful of newly stabilized APIs.
+## Compatibility notcie
+
+This is a first *minor* release after the initial release of Scala 3.0. This has following consequences:
+
+- Scala 3.1 is backward source compatible: any code that was working in 3.0 will also work in 3.1.
+- Scala 3.1 is backward binary compatible: you can use dependencies compiled with Scala 3.0 in 3.1 projects.
+- Scala 3.1 is _not_ forward binary compatible: you _cannot_ use dependencies compiled with Scala 3.1 in Scala 3.0 projects.
+- Updating Scala to 3.1 is not forward compatible change. If you are a library maintainer you should only do that in minor release of your project.
```

Comment:
This seems to be making some specific assumptions about what minor releases of projects should contain and the Scala.js release notes don't say anything like that /cc @sjrd

Reply:
Actually I disagree with your comments. A library that follows semver cannot update to `3.1` in a patch release.

Reply:
> Actually I disagree with your comments. A library that follows semver cannot update to `3.1` in a patch release.

Irrespective of that, this is not the place to assume that library authors are using semver, or any other scheme for that matter. They need to do the right thing for their own versioning scheme.

Reply:
s/many/may/